### DISCUSSION TITLE
Uh...  Can't you just add the XML to the ISO and then add the ISO to Ventoy?

### DISCUSSION THREAD

**Discussion by leeuniverse** [memstechtips/UnattendedWinstall#149] [General]
So, Ventoy works by just adding any (most) boot ready ISO to the Ventoy Partition for ISO's
So, can't you just add the XML to the Windows ISO using whatever your ISO tool is, and then just add the ISO to Ventoy?

I mean, if the XML automatically is read it should work with a boot ready ISO too right?

**UPDATE:**
Answered my own question....  Just saw this video of yours "How to Use Unattend.xml Files During Windows Installation (UnattendedWinstall Tutorial)"
And you DO show just adding the ISO to Ventoy, no complicated creating Template and ISO folders etc.

**Comment by amordesoul** [comment]
The easiest solution is to let the windows media creation app create the usb then copy the .xml file to the root of the USB after the app finishes.

**Comment by Rabtos** [comment]
I don't know why this simple Method with basicly only 1 Step is hidden away. It should be at the START of the Readme ..

**Comment by wagner2029** [comment]
I did this with the free version of Anyburn, go to edit ISO, load the XML and done.
https://www.anyburn.com/download.php
<img width="574" alt="image" src="https://github.com/user-attachments/assets/fd6c3975-7f2a-4c17-8bfc-9baaf403f5a3" />
